.github/workflows/tests.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           print("::set-output name=result::{}".format(result))
 
       - name: Restore pre-commit cache
-        uses: actions/cache@v4.0.2
+        uses: actions/cache@v4
```

Comment:
Please undo this - it is managed by dependabot
I will get those updated in parallel